feeds/ro.json

```diff
@@ -25,34 +25,52 @@
         {
             "name": "Brasov",
             "type": "mobility-database",
-            "mdb-id": "mdb-2143"
+            "mdb-id": "mdb-2143",
+            "license": {
+            "spdx-identifier": "ODbL-1.0"
+            }
         },
         {
             "name": "Bucuresti-Ilfov",
             "type": "mobility-database",
-            "mdb-id": "mdb-2098"
+            "mdb-id": "mdb-2098",
+            "license": {
+            "spdx-identifier": "CC0-1.0"
+            }
         },
         {
             "name": "Bucuresti-Ilfov",
             "type": "mobility-database",
             "spec": "gtfs-rt",
-            "mdb-id": "mdb-2103"
+            "mdb-id": "mdb-2103",
+            "license": {
+            "spdx-identifier": "CC0-1.0"
+            }
         },
         {
             "name": "Cluj-Napoca",
             "type": "mobility-database",
-            "mdb-id": "mdb-2121"
+            "mdb-id": "mdb-2121",
+            "license": {
+                "spdx-identifier": "CC-BY-SA-4.0"
+            }
         },
         {
             "name": "Cluj-Napoca",
             "type": "mobility-database",
             "spec": "gtfs-rt",
-            "mdb-id": "mdb-2612"
+            "mdb-id": "mdb-2612",
+            "license": {
+                "spdx-identifier": "CC-BY-SA-4.0"
+            }
         },
         {
             "name": "Consiliul-Judetean-Vrancea",
             "type": "mobility-database",
-            "mdb-id": "mdb-1984"
+            "mdb-id": "mdb-1984",
+            "license": {
+            "spdx-identifier": "CC0-1.0"
```

Comment:
formatting (you can use the bulk-edit-feeds.py script for automatic formatting. It's just a side effect but it works)